app/assets/javascripts/tasks_ratings.coffee

```diff
@@ -2,48 +2,42 @@ ready = ->
   initializeRatings()
 
 initializeRatings = ->
-  $('.rating span').hover (->
-    $(this).removeClass("fa-regular").addClass("fa-solid")
-    rating = this.getAttribute("data-rating")
-    lower = $('.rating span').filter (->
-      $(this).attr("data-rating") < rating
-    )
-    $(lower).removeClass("fa-regular").addClass("fa-solid")
-    upper = $('.rating span').filter (->
-      $(this).attr("data-rating") > rating
-    )
-    $(upper).removeClass("fa-solid").addClass("fa-regular")
+  $(".task-star-rating[data-is-rating-input='true'] .rating-star").hover(->
```

Comment:
I actually liked the hover mechanism because it is faster but I can see why this could also be annoying. I switched to onclick :+1:

Reply:
Thanks for updating; I think clicking is better to confirm a choice. Still, I liked the hover *effect* to "preview" what my click would do. Can we re-add that in a way that the left three stars are dark gray if I hover over the third and get all white again if I move out of the rating area without clicking?

Reply:
I added a hover effect. Not sure if that is exactly what you had in mind but I think it looks good.

Reply:
Yes, that's what I intended and indeed, it looks great!

There is just a minor visual issue I noticed: Hovering over the last star somewhat changes the spacing of all (other) items. Since I cannot explain it with words, here's a screencast:

https://github.com/user-attachments/assets/527b3408-86ab-44db-9e47-53ef06830298

Reply:
This seems to be caused by a size difference between the filled stars and the empty stars. I changed some CSS and it works (on my machine) now.

Reply:
Yes, I can confirm it works on my machine, too 🎉. Thank you!